WebRoot/css/zm.css

```diff
@@ -1985,123 +1985,193 @@ div.CompDragArea * {
+	@FullWidth@
 }
 
-.newTaskBannerSep {
-	border-bottom:2px solid #AAA;
+.ZmContactHeader .ZmContactHeaderRow {
+	@ContactHeaderRow@
 }
 
-.ZmEditContactView .contactHeaderRow {
-	@PageBg@
+.ZmContactHeaderRow > DIV {
+	@ContactHeaderCell@
 }
 
-.ZmContactView .contactContentTable {
-	width: 100%;
+.ZmContactHeaderRow .ZmContactHeaderMain {
+	@ContactHeaderMain@
 }
 
-.ZmContactView .headerRow .rowLabel,
-.ZmContactView .headerRow .rowType {
-	width: 1%;
+.ZmContactHeaderMain .ZmContactHeaderImage {
+	@ContactHeaderImageWrapper@
 }
 
-.ZmContactView .headerRow .rowType > DIV {
-	@BoxMargin@
+.ZmContactHeaderImage IMG,
+.ZmContactHeaderImage .svg-icon {
+	@ContactHeaderImage@
 }
 
-.ZmContactView .headerRow,
-.ZmContactInfoView .headerRow {
-	background-color:@darken(AppC,5)@;
-	border-bottom:1px solid @darken(AppC,50)@;
+.ZmContactHeaderMain .ZmContactHeaderInfo {
+	@ContactHeaderInfo@
 }
 
-.ZmContactView .headerRowImage,
-.ZmContactInfoView .headerRowImage {
-	vertical-align:top;
+.ZmContactHeaderInfo .ZmContactTitle:not(:empty) {
+	@ContactHeaderTitle@
 }
 
-.ZmContactView .headerRowImage IMG,
-.ZmContactInfoView .headerRowImage IMG,
-.ZmContactView .headerRowImage DIV,
-.ZmContactInfoView .headerRowImage DIV {
-	@ContactImageRadius@
+.ZmContactHeaderInfo .ZmContactSubTitle:not(:empty) {
+	@ContactHeaderSubTitle@
 }
 
-.ZmContactView .contactHeaderSubTable,
-.ZmContactInfoView .contactHeaderSubTable {
-	background-color:@darken(AppC,15)@;
-	border-bottom:1px solid @darken(AppC,50)@;
+.ZmContactHeaderInfo .ZmContactHeaderText:not(:empty) {
+	@ContactHeaderText@
 }
 
-.ZmContactInfoView .contactGroupTable {
-	background-color:white;
-	border:1px solid @darken(AppC,50)@;
-	float:left;
-	margin:.5em;
-	width:30%;
+.ZmContactHeaderInfo .ZmContactHeaderJobInfo:not(:empty) {
+	@ContactHeaderJobInfo@
 }
 
-.contactHeaderTags {
-	padding-top: .5em;
+.ZmContactHeaderInfo .ZmContactHeaderTags:not(:empty) {
+	@ContactHeaderTags@
 }
 
-.contactGroupTableImage {
-	vertical-align:top;
-	text-align:center;
-	padding:1em 5px 0em;
-	height:70px;
+.ZmContactHeaderRow .ZmContactHeaderLocation {
+	@ContactHeaderLocation@
 }
 
-.contactGroupTableImage IMG {
-	@ContactGroupImageRadius@
+.ZmContactHeaderLocation > TABLE {
+	@ContactHeaderLocationTable@
 }
 
-.contactGroupTableContent {
-	padding-right:.5em;
+.ZmContactHeaderLocation .ZmContactHeaderLocationIcon {
+	@ContactHeaderLocationIcon@
 }
 
-.contactGroupTableContent SPAN {
-	@NoWrap@
+.ZmContactHeaderLocation .ZmContactHeaderLocationName {
+	@ContactHeaderLocationName@
+}
+
+.ZmContactHeaderLocation .ContactHeaderLocationLabel {
+	@ContactHeaderLocationLabel@
 }
 
-.contactGroupTableContent .ImgPlus {
+/* Contact Info View Style */
+.ZmContactInfoView .ZmContactInfo:not(:empty) {
+	@ContactInfo@
+}
+
+.ZmContactInfo > TABLE {
+	@FullWidth@
+}
+
+.ZmContactInfo .ZmContactInfoMsg {
+	@ContactInfoMsg@
+}
+
+.ZmContactInfo .ZmContactInfoAction {
+	@ContactInfoAction@
+}
+
+/* Contact view info style */
```

Comment:
is it different from above styles? I see a similar comment at line 2069.

Contact info view style vs Contact view info style?